### DISCUSSION TITLE
only two columns are visible

### DISCUSSION THREAD

**Discussion by Geraldf** [Comcast/react-data-grid#3168] [Q&A]
I have the following code, it should show 3 columns, but only the first two are displayed. Any ideas, why the third is not shown?

`import React from "react";
import "./App.css";
import DataGrid from "react-data-grid";

const columns = [
  { key: "id", name: "ID" },
  { key: "title", name: "Title" },
  { key: "title2", name: "Title2" },
];

const rows = [
  { id: 0, title: "Example" },
  { id: 1, title: "Demo" },
  { id: 2, title: "Demo" },
  { id: 3, title: "Demo" },
  { id: 4, title: "Demo" },
];

function App() {
  return (
    <div>
      <DataGrid columns={columns} rows={rows} />;
    </div>
  );
}

export default App;
`

**Comment by beltrewilton** [comment]
title2 on rows is missing

```javascript
const rows = [
  { id: 0, title: "Example", title2: "my title 2" },
  { id: 1, title: "Demo", title2: "my title 2 again" },
  { id: 2, title: "Demo", title2: "my title 2 :) " },
  { id: 3, title: "Demo" , title2: "my ...... "},
  { id: 4, title: "Demo" , title2: "just my title 2"},
];
```
